src/login.coffee

```diff
@@ -35,6 +35,18 @@ define ['jquery', 'primedia_events', 'jquery.cookie'], ($, events) ->
 
         $("a.logout").click (e) => @_logOut e
 
+    _prefillAccountName: ($div) ->
```

Comment:
The lib is already so coupled to the app, it's going to take a lot of work to separate them. There are CSS selectors all over the place, and passing in the input selectors only moves the problem up a level. That, said, I'll change the code if you'd prefer :)